hail/python/hailtop/hailctl/dev/cli.py

```diff
@@ -12,11 +12,6 @@ def parser():
         description='Manage Hail development utilities.')
     subparsers = main_parser.add_subparsers()
 
-    subparsers.add_parser(
```

Comment:
I think it wouldn't be a bad idea to have `hailctl` be able to `exec` benchmarks as well.

Reply:
not opposed to adding this in the future, but I can't figure out the right way to make that work at the moment.

Reply:
That's fair. The functionality that we want would be keep the `hailctl dev benchmark` subcommand, but pass through all of the args to the `[os.execvp](https://docs.python.org/3/library/os.html?highlight=execvp#os.execvp)` function, to call `hail-bench`. I'm happy to add that back after this PR is in.